src/models/db-d2.ts

```diff
@@ -123,4 +139,46 @@ export default class DbD2 {
         console.log({ dashboardId: uid });
         return { id: uid };
     }
+
+    async createCharts(
+        name: String,
+        organisationUnits: OrganisationUnitPathOnly[],
+        antigens: CategoryOption[],
+        datasetId: String,
+        startDate: Date | null,
+        endDate: Date | null
+    ): Promise<Ref[]> {
+        const organisationUnitsIds = organisationUnits.map(ou => ({ id: ou.id }));
+        const organizationUnitsParents = organisationUnits.reduce(
+            (acc, ou) => ({ ...acc, [ou.id]: ou.path }),
+            {}
+        );
```

Comment:
Minor, but that's a useful pattern: `const organizationUnitsParents = _(organisationUnits).map(ou => [ou.id, ou.path]).fromPairs().value();`